content/why-keptn/tool-integrations-right.md

```diff
@@ -2,4 +2,4 @@
 
 Keptn integrations translate well-defined CloudEvents into proprietary vendor APIs and hide complex automation for advanced tasks. 
 
-This makes exchanging tools a simple configuration change rather than touching each individual pipeline.  
+This makes exchanging tools a simple configuration change rather than touching each individual pipeline. This means that you can use Keptn to orchestrate any tools you like, for more tools check out [Keptn and other tools](https://keptn.sh/docs/concepts/keptn-tools/)
```

Comment:
```suggestion
This makes exchanging tools a simple configuration change rather than touching each individual pipeline. This means that you can use Keptn to orchestrate any tools you like, for more information about integrating tools , check out [Keptn and other tools](https://keptn.sh/docs/concepts/keptn-tools/).
```

Reply:
Updated :)